SPECS/igc-module.spec

```diff
@@ -1,13 +1,19 @@
+%define vendor_name Intel
+%define vendor_label intel
+%define driver_name igc
```

Comment:
Are they required? Citrix adds these in their vendor drivers, but we don't use them in this spec file.

Reply:
No, not required.